charts/add-ons/grafana/values.yaml

```diff
@@ -0,0 +1,2 @@
+image:
+  name: gcr.io/linkerd-io/grafana
```

Comment:
This is no longer used in the template, so might wanna remove it here?

Reply:
Sorry, I confused the name of the image with the resource name.

Reply:
OTOH it looks like `image.tag` is missing here

Reply:
Oh I didn't see the nested `default` in the template. So `image.tag` will only be used to override the tag coming from the controller version or linkerd version. And it's already documented in both the readme file and the main values.yaml. Good :+1: